src/clr-angular/forms/combobox/option.ts

```diff
@@ -38,36 +63,48 @@ export class ClrOption<T> implements OnDestroy {
   }
 
   private initializeSubscription(): void {
-    this.subscription = this.optionSelectionService.valueChanged.subscribe((value: T) => {
-      // Check for null and undefined needed because if the user doesnt assign a value to the option,
-      // all options should not be selected as the value would be null or undefined
-      if (value === null || value === undefined) {
-        this.selected = false;
-      } else if (this.value === value) {
-        // TODO: Render option when current selection is set by the user
-        this.selected = true;
-      } else {
-        this.selected = false;
-      }
-    });
+    this.subscriptions.push(
+      this.optionSelectionService.selectionChanged.subscribe((value: ComboboxModel<T>) => {
```

Comment:
value is unused

Reply:
Yes, remained after some refactoring. Thanks, removed it.